unstructured/partition/html/transformations.py

```diff
@@ -437,6 +437,12 @@ def extract_tag_and_ontology_class_from_tag(
         html_tag = "span"
         element_class = ontology.UncategorizedText
 
+    # Scenario 5: Image with no text a ontology.UncategorizedText element_class
```

Comment:
wording here is a bit confusing. 
```suggestion
    # Scenario 5: UncategorizedText has image and no text
```

maybe?